src/crab_verifier.cpp

```diff
@@ -130,19 +122,16 @@ static void print_report(std::ostream& os, const checks_db& db, const Instructio
         for (const auto& msg : messages) {
             if (print_line_info) {
                 auto line_info = label_to_line_info.find(label.from);
-                if (line_info != label_to_line_info.end() && line_info->second.has_value())
-                    os << line_info->second.value();
+                if (line_info != label_to_line_info.end())
+                    os << line_info->second;
             }
             os << label << ": " << msg << "\n";
         }
     }
     os << "\n";
-    if (!db.maybe_nonterminating.empty()) {
-        os << "Could not prove termination on join into: ";
-        for (const label_t& label : db.maybe_nonterminating) {
-            os << label << ", ";
-        }
-        os << "\n";
+    constexpr int max_instructions = 100000;
```

Comment:
FYI for comparison, I heard one of the linux maintainers say the max instructions for that verifier is 1 million

Reply:
"instructions" is now a misnomer, as it only counts iterations. It will probably end up as a parameter.